### PR TITLE
bpo-31201: Clarify command to re-run failing test(s) with example

### PR DESCRIPTION
**Pull request by aaronang**
https://bugs.python.org/issue31201

https://bugs.python.org/issue31201

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: README.rst [hunk 1]
```diff
@@ -172,9 +172,10 @@ something is wrong.
 By default, tests are prevented from overusing resources like disk space and
 memory.  To enable these tests, run ``make testall``.
 
-If any tests fail, you can re-run the failing test(s) in verbose mode::
+If any tests fail, you can re-run the failing test(s) in verbose mode.  For
+example, if ``test_os`` and ``test_gdb`` failed, you can run::
 
-    make test TESTOPTS="-v test_that_failed"
+     make test TESTOPTS="-v test_os test_gdb"
```
Comment (ned-deily) [inline README.rst]: Why the (additional) preceding space?

**Review by ned-deily — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (aaronang) [reply README.rst]: Oops, nice catch.

**Comment by aaronang** [comment]
I have made the requested changes; please review again

**Review by ned-deily — APPROVED** [review]

**Pull request merged**